### PR TITLE
fix: adding max-height and inner-scroll to chart config panel

### PR DESCRIPTION
**Pull request by yasmin-e**
Closes:  [#4127](https://github.com/lightdash/lightdash/issues/4127)

### Description:
image

### CODE REVIEW

**Comment by PriPatel** [comment]
I think the max-height could do with being bigger. The panel can be as tall as it needs to be so long as it isn't as long as the page (like in the crazy original screenshot) - wdyt? I think this is for cases where there are crazy amounts of series'. 

I found this bit surprising and sometimes it might not be obvious that you have to scroll! It still wouldn't have taken up the full page and it would have been nice to see everything.
Screenshot 2023-01-24 at 20 20 55

File: packages/frontend/src/components/ChartConfigPanel/ChartConfigPanel.styles.ts [hunk 1]
```diff
@@ -20,6 +21,8 @@ export const InputWrapper = styled(FormGroup)`
 
 export const Wrapper = styled.div`
     max-width: 28.571em;
+    max-height: 35em;
```
Comment (IrakliJani) [inline packages/frontend/src/components/ChartConfigPanel/ChartConfigPanel.styles.ts:24]: out of scope but we should stop using em units. they are really confusing 🥲

**Review by IrakliJani — APPROVED** [review]

**Review by rephus — APPROVED** [review]
Tested, it is working fine. just 1 suggestion, should we leave the tabs at the top ? (just scroll the tab content) , unless it is too hard because it then needs be done on every component... @PriPatel @IrakliJani 

image

**Comment by PriPatel** [comment]
Agree @rephus - I'll open a new ticket for pinning the tabs!

**Pull request merged**